content/tips-on-applying-for-teacher-training.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Tips on applying for teacher training
+description: |-
+date: "2021-04-20"
+keywords: []
+---
+
+Give yourself the best chance of getting on the course you want.
+
+Discover tips on writing your application, finding referees and preparing for
+interviews.
+
+## Browse courses
+
+You can [search for most postgraduate teacher training courses on GOV.UK](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+
+You can apply to up to 3 courses in total, and then accept one offer.
+
+Many people do not get in the first time, but there’s no need to wait until the
+next year to apply again.
+
+You can rework your application and apply to further courses one at a time.
+
+[Attend an event](/events) to get your course questions answered.
+
+If you do not find the right course for you on GOV.UK, there are [other ways to
+train](/ways-to-train).
+
+## Questions to ask your training provider
+
+Every training provider and training programme is different.
+
+Note down any questions that come to mind while browsing courses.
+
+You may want to ask potential training providers questions, such as:
+
+1. How are academic work and practical teaching arranged?
```

Comment:
do we want this to be a numbered list? Or just bullet points? I don't think i've seen this style of numbering anywhere else on site

Reply:
Good question. I don't think the order matters. Thoughts @EmmaFrith?

Reply:
I don't think questions lend themselves well to bullets - normally bullets are for v short, sharp bits of content with no capitalisation or punctuation - hence the numbered list.